lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -102,6 +102,15 @@ def script
               echo "rsync --delete -av "${RSYNC_PROTECT_TMP_FILES[@]}" --filter \\"- CVS/\\" --filter \\"- .svn/\\" --filter \\"- .git/\\" --filter \\"- .hg/\\" --filter \\"- Headers\\" --filter \\"- PrivateHeaders\\" --filter \\"- Modules\\" \\"${source}\\" \\"${DWARF_DSYM_FOLDER_PATH}\\""
               rsync --delete -av "${RSYNC_PROTECT_TMP_FILES[@]}" --filter "- CVS/" --filter "- .svn/" --filter "- .git/" --filter "- .hg/" --filter "- Headers" --filter "- PrivateHeaders" --filter "- Modules" "${source}" "${DWARF_DSYM_FOLDER_PATH}"
             fi
+
+            local basename
+            basename="$(basename -s .framework.dSYM "$source")"
+            binary="${DWARF_DSYM_FOLDER_PATH}/${basename}.framework.dSYM/Contents/Resources/DWARF/${basename}"
+
+            # Strip invalid architectures so "fat" simulator / device frameworks work on device
```

Comment:
Should the comment say "Strip invalid dSYMs" instead of "architectures"?

Reply:
I think it's fine, it is actually stripping architectures :)